multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -465,19 +471,36 @@ def best_quality_ner():
 @automm_presets.register()
 def medium_quality_faster_inference_ner():
     return {
-        "model.names": ["ner_text"],
+        "model.names": [
+            "categorical_mlp",
+            "numerical_mlp",
+            "timm_image",
+            "ner_text",
+            "fusion_ner",
+        ],
         "model.ner_text.checkpoint_name": "google/electra-small-discriminator",
     }
 
 
 @automm_presets.register()
 def high_quality_fast_inference_ner():
     return {
-        "model.names": ["ner_text"],
+        "model.names": [
+            "categorical_mlp",
+            "numerical_mlp",
+            "timm_image",
+            "ner_text",
+            "fusion_ner",
+        ],
         "model.ner_text.checkpoint_name": "microsoft/deberta-v3-base",
     }
 
 
+@automm_presets.register()
+def ner():
```

Comment:
Shall we call it `default_ner`?

Reply:
Currently, all specific problem types, e.g., `image_text_similarity` and `object_detection`, have registered presets. `default` is for the general problem types `binary`, `multiclass`, and `regression`.